solr/core/src/test/org/apache/solr/search/QueryEqualityTest.java

```diff
@@ -1351,13 +1351,13 @@ protected void assertQueryEquals(
       SolrRequestInfo.clearRequestInfo();
     }
 
-    for (int i = 0; i < queries.length; i++) {
-      QueryUtils.check(queries[i]);
+    for (Query value : queries) {
+      QueryUtils.check(value);
       // yes starting j=0 is redundent, we're making sure every query
       // is equal to itself, and that the quality checks work regardless
       // of which caller/callee is used.
-      for (int j = 0; j < queries.length; j++) {
-        QueryUtils.checkEqual(queries[i], queries[j]);
+      for (Query query : queries) {
+        QueryUtils.checkEqual(value, query);
```

Comment:
How about naming to indicate that both are from `queries` ultimately? E.g.
```suggestion
        QueryUtils.checkEqual(query1, query2);
```